packages/design-system/src/components/N8nBlockUi/BlockUi.vue

```diff
@@ -0,0 +1,44 @@
+<template>
+	<transition name="fade" mode="out-in">
+		<div v-show="show" :class="['n8n-block-ui', $style.uiBlocker]" role="dialog" :aria-hidden="true" />
+	</transition>
+</template>
+
+<script lang="ts">
```

Comment:
nitpick - but we should move towards composition api for new components.. no need to update this for now.. just something to keep in mind

Reply:
done, this component has only 1 prop so it even got less lines of code in no time (refactoring took 1 minute)